group/curve25519/ext.go

```diff
@@ -83,11 +81,11 @@ func (P *extPoint) HideDecode(rep []byte) {
 //		iff
 //	(X1*Z2,Y1*Z2) == (X2*Z1,Y2*Z1)
 //
-func (P1 *extPoint) Equal(CP2 kyber.Point) bool {
+func (P *extPoint) Equal(CP2 kyber.Point) bool {
 	P2 := CP2.(*extPoint)
 	var t1, t2 mod.Int
-	xeq := t1.Mul(&P1.X, &P2.Z).Equal(t2.Mul(&P2.X, &P1.Z))
-	yeq := t1.Mul(&P1.Y, &P2.Z).Equal(t2.Mul(&P2.Y, &P1.Z))
+	xeq := t1.Mul(&P.X, &P2.Z).Equal(t2.Mul(&P2.X, &P.Z))
```

Comment:
Question: does this modify `P.X`? If so, then this is a race condition!

Reply:
From a quick look, it seems so. Would it be okay to fix that in a different PR ? I only renamed here to fix the lint warning.